src/AbstractWrapperCachingEventManager.php

```diff
@@ -37,6 +37,22 @@ protected function _getHandlerWrapper($name, $callback)
     }
 
     /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     *
+     * @return AbstractWpEventManager This instance.
+     */
+    protected function _detach($event, $callback, $priority = self::DEFAULT_PRIORITY)
```

Comment:
Excellent, fixes the inability to detach handlers. Good use of `_getHandlerWrapper()`.